addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/handler/BigAssFanHandler.java

```diff
@@ -405,19 +471,26 @@ private PercentType convertLevelToPercent(String level) {
         return new PercentType((int) Math.round(Integer.parseInt(level) * BRIGHTNESS_CONVERSION_FACTOR));
     }
 
+    private boolean isLightPresent() {
+        if ("PRESENT".equals(fanStateMap.get(CHANNEL_LIGHT_PRESENT).toString())) {
```

Comment:
I would say that this comparison is fishy, what is the actual type you get from the fanStateMap, calling `toString` on something to compare the string representation is most of the time not the most stable solution.

Also please return a Boolean directly instead of feeding it to an if to convert it to a boolean 🙂